server.js

```diff
@@ -4954,13 +4954,17 @@ cache(function(data, match, sendBadge, request) {
           break;
         case 'testing':
         case 'waiting':
+        case 'initiated':
           badgeData.text[1] = 'pending';
           break;
         case 'error':
+        case 'infrastructure_failure':
```

Comment:
Not sure if we should set this as `error` or `failing`, what do you think is the best text to describe this.

Codeship returns `error` for this status maybe we should follow suit?
![image](https://user-images.githubusercontent.com/7288322/39335078-a3d76cf2-4a05-11e8-9fd3-72fc46b03e4a.png)

Reply:
I don't feel the need to add a new status, if there is an error with the build itself, that's a failing build all the same imo.